include/c_common/pgr_point_input.h

```diff
@@ -29,8 +29,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 /* for size-t */
 #include <stddef.h>
 
-#include "c_types/pgr_point_t.h"
-
+typedef struct Pgr_point_input_t Pgr_point_input_t;
```

Comment:
The actions will point if it is wrong (by sight it is), probably you need to change to
```
typedef struct Pgr_point_t Pgr_point_t;
```
btw, maximum 2 commits.
```
git fetch upstream
git rebase -i upstream/develop
```
Use fixup, to put the extra commits into one (help shows on the same file it opens)